rest-api/tools/consent_upload_validation.py

```diff
@@ -0,0 +1,139 @@
+""" read from csv (assumes participant id column header is [pmi_id] and paired site header is
+    [paired_site]).
+     Checks PTC-UPLOADS-ALL-OF-US-RDR-PROD bucket and the bucket given in args (i.e. aouXXX)
+     Compares PTC uploaded files (consent pdf's and signature png's) and files located in awardee
```

Comment:
This comment is helpful. What kind of files are we expecting to find in the awardee bucket?

Reply:
elaborated comment.